docs/getting-started/index.html

```diff
@@ -4,19 +4,55 @@
 group: "navigation"
 ---
 <div>
-	<h2>Getting Started</h2>
-	<p>Below you will find various materials to give you an introduction to the W3C Web of Things and its technology.</p>
+	<h2>Introduction</h2>
+	<p>The W3C Web of Things (WoT) seeks to counter the fragmentation of the IoT, making it much easier to create 
+		applications without the need to master the disparate variety of IoT technologies and standards. 
+		To realize this, WoT specifies some principles and technology building blocks that can be used to 
+		complement new or existing IoT ecosystems. An overview of the main WoT principles and building 
+		blocks is given here:</p>
+	<h3>Thing</h3>
+	<p>A Thing is an abstraction of a physical or virtual entity that provides interactions to and participates 
+		in the Web of Things. </p>
+	<h3>Interaction Affordances: Properties, Actions, Events</h3>
+	<p>The Interaction Model of W3C WoT introduces an intermediate abstraction that formalizes the mapping from application 
+		intent to concrete protocol operations and also narrows the possibilities how Interaction Affordances can be modeled.
+		So, Things may offer three other types of Interaction Affordances defined by this specification: 
+		Properties, Actions, and Events. 
+	</p>
```

Comment:
<h3>Consumer</h3>

A consumer of a thing is able to parse and process the Thing Description to be able to interact with the thing. This interaction can be over a direct connection or via one or more intermediaries.

<h3>Intermediary</h3>

Intermediaries can act as proxies for Things, where the Intermediary has a WoT Thing Description similar to the original Thing, but which points to the WoT Interface provided by the Intermediary. Intermediaries may also augment existing Things with additional capabilities or compose a new Thing out of multiple available Things, thereby forming a virtual entity. To Consumers, Intermediaries look like Things, as they possess WoT Thing Descriptions and provide a WoT Interface.